spec/parliament/response_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../spec_helper'
+require 'pry'
+
+describe Parliament::Response, vcr: true do
+  let(:nodes) { [] }
+  subject { Parliament::Response.new(nodes) }
+
+  describe '#initialize' do
+    it 'sets an instance variable for the nodes' do
+      expect(subject.instance_variable_get(:@nodes)).to eq(nodes)
+    end
+  end
+
+  describe '#filter' do
```

Comment:
Should there be a test where you call filter on an empty `Parliament::Response` object?